ui/BackendLocal.qml

```diff
@@ -30,196 +31,239 @@ Item {
 
     ListModel {
         id: backendFeatureList
+
         ListElement {
-            text: "Automatic Pairing"
+            text: QT_TR_NOOP("Automatic Pairing")
         }
         ListElement {
-            text: "Needs to be hosted in a private network before hand"
+            text: QT_TR_NOOP("Needs to be hosted in a private network before hand")
         }
         ListElement {
-            text: "Configure STT in backend"
+            text: QT_TR_NOOP("Configure STT in backend")
         }
         ListElement {
-            text: "Configurable TTS Options: Mimic2 | Mimic | Larynx | Pico"
+            text: QT_TR_NOOP("Configurable TTS Options: Mimic2 | Mimic | Larynx | Pico")
```

Comment:
this needs to be updated, if we make plugins per lang then this should not contain hardcoded names anymore, same for all language files

Reply:
fixed removed the hardcoded names